tests/unit/lms/services/blackboard_api/client_test.py

```diff
@@ -70,7 +71,10 @@ def test_it_with_pagination(self, svc, basic_client, blackboard_list_files_schem
 
         # It called the Blackboard API three times getting the three pages.
         assert basic_client.request.call_args_list == [
-            call("GET", "courses/uuid:COURSE_ID/resources?type=file&limit=200"),
+            call(
+                "GET",
+                "courses/uuid:COURSE_ID/resources?type=file&limit=200&fields=id,name,modified,mimeType",
+            ),
```

Comment:
Could be a good place for `Any.url` so we don't mind about changes like argument order and encoding.

Reply:
This does work but it reads a bit misleadingly because the method doesn't actually accept URLs, it takes paths